src/main/java/com/amazon/opendistroforelasticsearch/ad/settings/AnomalyDetectorSettings.java

```diff
@@ -347,6 +347,7 @@ private AnomalyDetectorSettings() {}
 
     public static int THRESHOLD_MODEL_TRAINING_SIZE = 1000;
 
+    public static int MAX_OLD_AD_TASK_DOCS = 1000;
```

Comment:
Could you add some comments for the setting? There are so many settings, some of them are self explained by the name, but some of them doesn't have clear name.

Reply:
sure